docs/source/traits_user_manual/defining.rst

```diff
@@ -278,7 +278,9 @@ the table.
 +------------------+----------------------------------------------------------+
 | Name             | Callable Signature                                       |
 +==================+==========================================================+
-| Any              | Any( [*value* = None, \*\*\ *metadata*] )                |
+| Any              | Any( [*default_value* = None, \*,                        |
+|                  | *factory* = None, *args* = (), *kw = {},                 |
```

Comment:
```suggestion
|                  | *factory* = None, *args* = (), *kw* = {},                 |
```

we might be missing something here because this is how it looks in the built docs -
`Any( [default_value = None, *, factory = None, args = (), kw = {}, ***metadata )`

Reply:
Thanks. I _did_ build the docs, but failed to spot this.